packages/babel-plugin-proposal-json-modules/test/fixtures/browser/import-expression/output.mjs

```diff
@@ -0,0 +1,6 @@
+// TODO: Not supported yet
```

Comment:
I'm not sure how to support this, given that:
- if it's not a JSON import, we need to fallback to an import with two arguments
- that is not known statically, so we need to always emit code for that fallback
- the browser might only support import with a single argument

I'd not block this feature on solving that problem, but rather document it as a known limitation.